tools/src/wyvern/tools/tests/modules/arrowModuleClient2.wyv

```diff
@@ -0,0 +1,5 @@
+require stdout
```

Comment:
`stdout` isn't used. Is there a reason why it is imported in this file?

Reply:
Hi Darya,

stdout is imported by mistake. I have omitted the import statement. Thank you for pointing it out!

Simon